sdk/rust/macro/wit/reactor.wit

```diff
@@ -0,0 +1,6 @@
+package fermyon:spin
+
+world exports {
```

Comment:
Nothing actually exports both inbound-http and inbound-redis does it?  I recall something about this causing problems in custom triggers where the SDK wouldn't work unless you faked up an export... is this an appropriate time to address that?  (Or right time but wrong PR? _grin_)

Reply:
I was able to fix this 🎉